### ISSUE TITLE
Color-Blind friendly UI

### ISSUE THREAD

**Issue by sparakala21** [Bram-Hub/LEGUP#426]
### Describe the current behavior of what you're trying to improve. If your enhancement request related to a problem, please also describe the problem.

Lots of LEGUP's UI is based on things lighting up red or green. The problem with that is that color blind people(like myself) have a hard time understanding when something is wrong. 

### Describe the improvement you'd like

Simple fixes can include changing the contrast between the reds and greens, or switching to red and blue.(which i think would be more compatible with more types of color-blindness)

**Comment by charlestian23** [comment]
Yeah, this change would be really nice (would help me as well). I think the best thing to do is to change green to blue, but that's just my suggestion.

**Comment by N-Desmarais** [comment]
The ADA recommends against the following color combinations:
    Green – Blue
    Green – Yellow
    Green – Red
    Blue – Purple
    Blue – Grey
    Green – Brown
    Green – Grey
Red & Blue might be a slight upgrade or maybe allow users to specify their own colors

**Comment by sparakala21** [comment]
That sounds like a good idea. do you want to work on this together?

**Comment by charlestian23** [comment]
Specifying own colors would be a nice feature, though if that is too much work for you, we can stick to changing all instances of green to blue for now. Although, LEGUP might already have the color blue reserved for something, so you might have to pick a new color scheme. I'll leave the design up to you.

**Comment by N-Desmarais** [comment]
You can find the colors for the proof tree in:
src/main/java/edu/rpi/legup/ui/proofeditorui/treeview/TreeTransitionView.java

The colors you should care about are CORRECT_COLOR and INCORRECT_COLOR, but you may want to play around with others to make things look nice

**Comment by Mystin** [comment]
We now have a system for customizing UI colors using FlatLAF and .properties files. Basically all that's left to do for full integration is creating color-changing svg rule icons, which coincides with #525 .
